services/imagery/src/image-store.js

```diff
@@ -60,37 +114,71 @@ export default class ImageStore extends EventEmitter {
    * The metadata attached is the Image proto message without the
    * images included.
    *
-   * Returns the id number for the image (the first one is 0).
+   * Returns the id number for the image (the first one is 1).
```

Comment:
Can the default value be restored to 0 to keep the numbering consistent with the previous iteration.